src/containers/Login.js

```diff
@@ -1,17 +1,37 @@
 import React, { useState } from "react";
 import { Button, FormGroup, FormControl, ControlLabel } from "react-bootstrap";
 import "./Login.css";
+import Credentials from "../data/credentials.json";
 
 export default function Login(props) {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  var userFound = "no";
 
   function validateForm() {
     return email.length > 0 && password.length > 0;
   }
 
-  function handleSubmit(event) {
-    event.preventDefault();
+  async function handleSubmit(event) {
+    try {
+      if (email && password) {
+        Credentials.map((credentialData, index) => {
+          if (credentialData.email === email && credentialData.password === password) {
+            event.preventDefault();
+            userFound = "yes";
+            alert("Logged in");
+          }
+          return null;
+        })
+        if (userFound === "no") {
+          alert("Wrong email or password");
+        }
+      } else {
+        alert("Enter email and password");
```

Comment:
Do not use alert some own react component

Reply:
It is temporary.